tools/format.py

```diff
@@ -56,7 +56,10 @@ def gn_format():
     print "gn format"
     exe = os.path.join(third_party_path, "depot_tools", "gn")
     source_files = git_ls_files(root_path, ["*.gn", "*.gni"])
-    run([exe, "format", "--"] + source_files, env=google_env(), quiet=True)
+    run([exe, "format", "--"] + source_files,
+        cwd=os.path.join(root_path, "core", "libdeno"),
```

Comment:
This isn’t correct. We need to either use absolute paths for the source files here or find out if we can specify the gn root for gn format somehow and drop the cws change here.

Will look into it once I arrive home.